icicle/include/icicle/utils/modifiers.h

```diff
@@ -26,3 +26,9 @@
 #define __host__
 #define __device__
 #endif
```

Comment:
we are not compatible with windows anyway.

Reply:
I know but the source was like that. Do you prefer deleting it? no way we support windows?